cmd/gh/main.go

```diff
@@ -99,10 +99,8 @@ func mainRun() exitCode {
 		return exitError
 	}
 
-	// TODO: remove after FromFullName has been revisited
-	if host, err := cfg.DefaultHost(); err == nil {
-		ghrepo.SetDefaultHost(host)
-	}
+	host, _ := cfg.DefaultHost()
+	ghrepo.SetDefaultHost(host)
```

Comment:
After this change we will be able to remove this and use `ghAuth.DefaultHost()` inside of `ghrepo` instead of the `defaultHostOverride` variable.

Reply:
O.k.